src/parser/events-assets/grammar.ne

```diff
@@ -1,11 +1,9 @@
 @preprocessor typescript
 
-@builtin "number.ne"
-@builtin "string.ne"
+@include "../common.ne"
```

Comment:
`common.ne` doesn't need to be built separately? just import?

Reply:
yes, `common.ne` will be included in the dictionary while building